dotcom-rendering/src/web/components/ArticleHeadline.tsx

```diff
@@ -731,7 +733,9 @@ export const ArticleHeadline = ({
 											color: ${isMatch
 												? palette.text.headlineWhenMatch
 												: palette.text.headline};
-											padding-bottom: ${space[9]}px;
+											padding-bottom: ${isMainMediaTest
```

Comment:
This is being checked with Design to see if we need to make this change as 32px is not source standard.

Reply:
design are happy to align with source here.